py3status/modules/whatismyip.py

```diff
@@ -97,7 +100,7 @@ def post_config_hook(self):
         # Backwards compatibility
         self.substitutions = {}
         if self.url_geo == URL_GEO_NEW_DEFAULT:
-            self.substitutions['country'] = 'country_name'
+            self.substitutions['country_code'] = 'country_iso'
```

Comment:
I took a quick glance. Is this a sub between old URL and new URL? And is this everything? I think `ipstack.com` added few new things so I don't know the answer.

Reply:
I have not try to use ipstack.com, would try to make a just working module without api key. Seems the only needed modification, maybe i miss one. (with ifconfig.co)